project/Dependencies.scala

```diff
@@ -44,20 +45,29 @@ object Dependencies {
 
   val oauth = Seq("oauth.signpost" % "signpost-core" % "2.1.1")
 
-  val cachecontrol = Seq("com.typesafe.play" %% "cachecontrol" % "2.2.0")
+  val cachecontrol = Seq(
+    "com.typesafe.play"      %% "cachecontrol"             % "2.2.0",
+    "org.scala-lang.modules" %% "scala-parser-combinators" % "2.1.0"
+  )
 
   val asyncHttpClient = Seq("org.asynchttpclient" % "async-http-client" % "2.12.3")
 
-  val akkaStreams = Seq("com.typesafe.akka" %% "akka-stream" % "2.6.18")
-  val akkaHttp    = Seq("com.typesafe.akka" %% "akka-http" % "10.2.7")
+  val akkaStreams = Seq(("com.typesafe.akka" %% "akka-stream" % "2.6.18").cross(CrossVersion.for3Use2_13))
```

Comment:
@cchantep @xuwei-k Actually akka 2.6.18 got published for Scala 3 already... Shouldn't we use that artifact? See https://repo1.maven.org/maven2/com/typesafe/akka/akka-stream_3/